reporter/html.js

```diff
@@ -779,18 +779,22 @@ export function escapeText( s ) {
 			if ( actual !== expected ) {
 
 				message += "<tr class='test-actual'><th>Result: </th><td><pre>" +
-				escapeText( actual ) + "</pre></td></tr>";
-
-			// Don't show diff if actual or expected are booleans
-				if ( !( /^(true|false)$/.test( actual ) ) &&
-					!( /^(true|false)$/.test( expected ) ) ) {
+					escapeText( actual ) + "</pre></td></tr>";
+
+				if ( "number" === typeof details.actual && "number" === typeof details.expected ) {
```

Comment:
why is the typeof in the right side? This feels inverted.

Btw, we also have a `core/utilities` module checking for objectType. typeof is pretty much ok for number types but this order is confusing.

Other question: should we also confirm that neither details.actual and details.expected are not NaN?

Otherwise, a couple lines below, `diff = ( details.actual - details.expected );`, diff will be `NaN` as well.